api-conversion/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why do we need a licence file here?

Reply:
That was used to package the LICENSE with the ZIP / TAR archives. I removed it and copy the license instead from the root directory of the project.